modules/dhcp_common/isc/configuration_parser.rb

```diff
@@ -396,7 +427,7 @@ def visit_parse_tree_nodes(parse_tree, all_hosts, all_subnets, all_ignored, root
         end
 
         def parse_file(a_path)
-          File.open(a_path, 'r') {|f| conf.parse!(f.read, a_path)}
+          File.open(a_path, 'r:ASCII-8BIT') {|f| conf.parse!(f.read, a_path)}
```

Comment:
This is new to me, where can I find this in ruby docs?

Reply:
https://ruby-doc.org/core-2.0.0/Encoding.html, search for ascii-8bit